### PR TITLE
Add loader in sidepage search (without MainPage)

### PR DESCRIPTION
**Pull request by AtkishkinVlad**
Add loader in guides

### CODE REVIEW

File: components/Toolbar/MainPageToolbar.jsx [hunk 1]
```diff
@@ -40,9 +41,11 @@ export const MainPageToolbar = () => {
                 const responseJson = await response.json()
                 const { guideSuggestions } = responseJson
                 setGuideSuggestions(guideSuggestions)
+                setIsLoading(false)
```
Comment (fidlow) [inline components/Toolbar/MainPageToolbar.jsx:44]: where is declaration of this function?

File: components/Toolbar/MainPageToolbar.jsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export const MainPageToolbar = () => {
 
     const handleOnChange = useCallback(
         debounce(async (e) => {
+            loadingState.set(true)
```
Comment (fidlow) [inline components/Toolbar/MainPageToolbar.jsx:34]: where is import statement?

File: components/Toolbar/MainPageToolbar.jsx [hunk 3]
```diff
@@ -40,9 +41,11 @@ export const MainPageToolbar = () => {
                 const responseJson = await response.json()
                 const { guideSuggestions } = responseJson
                 setGuideSuggestions(guideSuggestions)
+                setIsLoading(false)
             } else {
                 setGuideSuggestions([])
             }
+            loadingState.set(false)
```
Comment (fidlow) [inline components/Toolbar/MainPageToolbar.jsx:48]: where is import statement?

File: components/SidePage/Suggestions/Suggestions.jsx [hunk 4]
```diff
@@ -97,12 +99,15 @@ export const Suggestions = ({ guideSuggestions }) => {
...
     return (
         <div className={styles.Suggestions__container}>
-            {guides?.map((guideSuggest) => (
-                <GuideSuggestion guideSuggestion={guideSuggest} />
-            ))}
+            {guideSuggestions.length > 0 ? (
+                guides?.map((guideSuggest) => <GuideSuggestion guideSuggestion={guideSuggest} />)
+            ) : currentQuery.length > 0 ? (
+                <p className={styles.Suggestions__not_found}>Ничего не нашли</p>
+            ) : null}
```
Comment (fidlow) [inline components/SidePage/Suggestions/Suggestions.jsx:110]:
```suggestion
            {guideSuggestions.length > 0 && (
                guides?.map((guideSuggest) => <GuideSuggestion guideSuggestion={guideSuggest} />}
            {guideSuggestions.length === 0 && currentQuery.length > 0 && (
                <p className={styles.Suggestions__not_found}>Ничего не нашли</p>
            )}
```

**Review by fidlow — CHANGES_REQUESTED** [review]

File: components/SidePage/Suggestions/Suggestions.jsx (hunk 4 above)
Comment (fidlow) [reply components/SidePage/Suggestions/Suggestions.jsx:110]: It's easier to read

File: components/Toolbar/MainPageToolbar.jsx (hunk 2 above)
Comment (AtkishkinVlad) [reply components/Toolbar/MainPageToolbar.jsx]: Fix

File: components/Toolbar/MainPageToolbar.jsx (hunk 1 above)
Comment (AtkishkinVlad) [reply components/Toolbar/MainPageToolbar.jsx]: Fix

File: components/Toolbar/MainPageToolbar.jsx (hunk 3 above)
Comment (AtkishkinVlad) [reply components/Toolbar/MainPageToolbar.jsx]: Fix

File: components/SidePage/Suggestions/Suggestions.jsx (hunk 4 above)
Comment (AtkishkinVlad) [reply components/SidePage/Suggestions/Suggestions.jsx]: Fix

**Review by fidlow — DISMISSED** [review]

**Comment by a-chabin** [comment]
Let's add styles to loader container
```
max-width: 60%;
aspect-ratio: 1 / 1;
width: 400px;
```

This should fix:
- Too big loader 
- Horizontal scroll

https://user-images.githubusercontent.com/22644149/224806542-8c4f3da9-4f3b-461a-bd12-6ce5c5781cbf.mov

**Comment by a-chabin** [comment]
Let's add a debounce decorator to optimize the network and fix the loader flashing

https://doka.guide/js/debounce/

File: components/Loader/Loader.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import styles from './loader.module.css'
+
+export function Loader({ isLoading }) {
+    return (
+        <div style={{ opacity: isLoading ? '1' : '0' }} className={styles.circles} id="circles">
+            <div className={`${styles.circle} ${styles.first}`} />
+            <div className={`${styles.circle} ${styles.second}`} />
+            <div className={`${styles.circle} ${styles.third}`} />
+            <div className={`${styles.logo}`}>
+                <svg
```
Comment (a-chabin) [inline components/Loader/Loader.jsx:12]: Let's get loader from package:

1. `npm i ekb`
2. `import { ProjectsPanel, PROJECT_GUIDES, Theme } from 'ekb'`

**Pull request merged**